ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,66 @@
+from ConfigParser import SafeConfigParser
+import logging
+import time
+import unittest
+
+logger = logging.getLogger('example')
+logging.basicConfig(level=logging.DEBUG)
+REFRESH_TIME = 15 * 60                           # 15 minutes
+
+
+class Config(object):
+
+    def __new__(cls, *args):
+        if not hasattr(cls, '_instance'):        # if there is not instance
```

Comment:
http://stackoverflow.com/questions/6760685/creating-a-singleton-in-python,  please read Answers 14: Use a Metaclass